src/main/scala/dpla/ingestion3/preMappingReports/XmlShredder.scala

```diff
@@ -168,13 +168,5 @@ object XmlShredder {
   def labelWithPrefix(node: Node): String =
     // We have to use null here b/c it can be returned by scala.xml.Node
     if(node.prefix == null) node.label
-    else s"${node.prefix}_${node.label}"
```

Comment:
I'm probably missing something glaring here but why was this changed from `_` to `.`.? Everything else seems to go the other way...

Reply:
Yeah, this concatenating a prefix with its label.  All the others are concatenating a parent label with a child label.  So there needs to be a different character here.